ObservatoryExplorer/DefaultCriteria.cs

```diff
@@ -61,13 +49,23 @@ public static List<(string Description, string Detail, bool SystemWide)> CheckIn
             #region Value Checks
             if (settings.HighValueMappable)
             {
-                if (HighValueTerraformablePlanetClasses.Contains(scan.PlanetClass) && scan.TerraformState?.Length > 0)
-                {
-                    results.Add("High-Value Mapping", $"{scan.DistanceFromArrivalLS:0}Ls, {scan.PlanetClass} (TF)");
-                }
-                if (HighValueNonTerraformablePlanetClasses.Contains(scan.PlanetClass) && scan.TerraformState?.Length == 0)
+                IList<string> HighValueNonTerraformablePlanetClasses = new string[] {
+                    "Earthlike body",
+                    "Ammonia world",
+                    "Water world",
+                };
+
+                if (HighValueNonTerraformablePlanetClasses.Contains(scan.PlanetClass) || scan.TerraformState?.Length > 0)
                 {
-                    results.Add("High-Value Mapping", $"{scan.DistanceFromArrivalLS:0}Ls, {scan.PlanetClass}");
+                    var info = new System.Text.StringBuilder(" ");
```

Comment:
After running a read-all on my journals, I noticed this is reporting well-known bodies in the bubble as FD:
![image](https://user-images.githubusercontent.com/54195004/168719692-dfc4737f-f246-4186-8d9a-666f05ec6c22.png)

If you look @ my custom criteria, I've got an outer "if not mapped" check that encloses both the FD/FM text generators. This is a "bug" in the journal data. I learned this trick to deal with it from Robby/Eahlstan from EDD.

Reply:
Good call about FD making FM redundant.

I abbreviated here because it was already getting kind of long, and many people aren't going to have a wide enough window to see it all. Or maybe _I_ just keep a very narrow window?

I'll rework this a bit to be more readable, and also filter out the undiscovered-in-bubble nonsense.

Reply:
FWIW, I have a wide but short window and it works well for me so in the end, lateral space isn't an issue. But maybe it's just me with 3 monitors and that much width to spare on a 4K display. :D